clients/m5stack-wired/extra_script.py

```
@@ -0,0 +1,15 @@
+Import("env")
```

Ruff (0.14.5)

Undefined name `Import`

(F821)